main.py

```diff
@@ -7,6 +7,6 @@
 
     # Set "use_ipython" to True to include the ipy command if IPython is installed, which supports advanced interactive
     # debugging of your application via introspection on self.
-    app = cmd2.Cmd(use_ipython=True)
+    app = cmd2.Cmd(use_ipython=True, persistent_history_file='cmd2_history.txt')
```

Comment:
FYI, there is an Issue that exists which recommends moving **main.py** underneath the **cmd2** directory and renaming it.  If you are making changes to it anyways, you might want to consider knocking that one out at the same time.